core/multimesh.f

```diff
@@ -522,3 +537,680 @@ subroutine neknek_xfer_fld(u,ui)
+        if (icvflow.eq.3) call cdtp(respr,h1,rzm2,szm2,tzm2,1)
+
+        dtbd = BD(1)/DT
+
+C     surface terms
+        DO 100 IEL=1,NELV
+          DO 300 IFC=1,2*ldim
+            CALL RZERO  (W1(1,IEL),NXYZ1)
+            CALL RZERO  (W2(1,IEL),NXYZ1)
+            IF (ldim.EQ.3)
+     $      CALL RZERO  (W3(1,IEL),NXYZ1)
+            CB = CBC(IFC,IEL,IFIELD)
+            IF (CB(1:1).EQ.'v'.and.intflag(ifc,iel).eq.1) then
+               CALL FACCL3
+     $         (W1(1,IEL),vxcbc(1,1,1,IEL),UNX(1,1,IFC,IEL),IFC)
+               CALL FACCL3
+     $         (W2(1,IEL),vycbc(1,1,1,IEL),UNY(1,1,IFC,IEL),IFC)
+               IF (ldim.EQ.3)
+     $          CALL FACCL3
+     $         (W3(1,IEL),vzcbc(1,1,1,IEL),UNZ(1,1,IFC,IEL),IFC)
+            ENDIF
+            CALL ADD2   (W1(1,IEL),W2(1,IEL),NXYZ1)
+            IF (ldim.EQ.3)
+     $      CALL ADD2   (W1(1,IEL),W3(1,IEL),NXYZ1)
+            CALL FACCL2 (W1(1,IEL),AREA(1,1,IFC,IEL),IFC)
+            IF (CB(1:1).EQ.'v'.and.intflag(ifc,iel).eq.1) then
+              CALL CMULT(W1(1,IEL),dtbd,NXYZ1)
+            endif
+            CALL SUB2 (RESPR(1,IEL),W1(1,IEL),NXYZ1)
+  300     CONTINUE
+  100   CONTINUE
+
+
+        call ortho    (respr)
+        call ctolspl  (tolspl,respr)
+
+        call hmholtz  ('PRES',prc,respr,h1,h2,pmask,vmult,
+     $                             imesh,tolspl,nmxh,1)
+        call ortho    (prc)
+
+C     Compute velocity
+ccccc
+        call opgrad   (resv1,resv2,resv3,prc)
+        if (ifaxis) call col2 (resv2,omask,n)
+        call opchsgn  (resv1,resv2,resv3)
+
+        if (icvflow.eq.1) call add2col2(resv1,v1mask,bm1,n) ! add forcing
+        if (icvflow.eq.2) call add2col2(resv2,v2mask,bm1,n)
+        if (icvflow.eq.3) call add2col2(resv3,v3mask,bm1,n)
+        if (ifexplvis) call split_vis ! split viscosity into exp/imp part
+
+        call copy(vxcp,vxc,lx1*ly1*lz1*nelv)
+        call copy(vycp,vyc,lx1*ly1*lz1*nelv)
+        call copy(vzcp,vzc,lx1*ly1*lz1*nelv)
+
+        intype = -1
+        call sethlm   (h1,h2,intype)
+        call ophx(resbc(1,1),resbc(1,2),resbc(1,3),
+     $             vxcbc,vycbc,vzcbc,h1,h2)
+        call sub2(resv1,resbc(1,1),n)
+        call sub2(resv2,resbc(1,2),n)
+        if (ldim.eq.3) call sub2(resv3,resbc(1,3),n)
+        call ophinv   (vxc,vyc,vzc,resv1,resv2,resv3,h1,h2,tolhv,nmxh)
+        call opadd2(vxc,vyc,vzc,vxcbc,vycbc,vzcbc)
+
+        call sub3(dvxc,vxcp,vxc,n)
+        call sub3(dvyc,vycp,vyc,n)
+        call sub3(dvzc,vzcp,vzc,n)
+        dvxmax = rs_glamax(dvxc,n)
+        dvymax = rs_glamax(dvyc,n)
+        dvzmax = rs_glamax(dvzc,n)
+        if (ifexplvis) call redo_split_vis ! restore vdiff
+        if (nid.eq.0)
+     $    write(6,'(i2,i8,i4,1p4e13.4,a11)') idsess,istep,ictr,time,
+     $    dvxmax,dvymax,dvzmax,' del-vol-vxy'
+
+      enddo
+
+      return
+      end
+c-----------------------------------------------------------------------
+      function rs_glmin(a,n)
```

Comment:
What's the difference to the existing glmin_ms()?

Reply:
These are not needed. I accidentally carried them over from my dev branch.